domFoundations/events,storage,attributes/storage.js

```diff
@@ -0,0 +1,8 @@
+export const saveData = (key, value) => {
+    localStorage.setItem(key, value);
+}
+
+export const loadData = (key) => {
+    const loadedData = localStorage.getItem(key);
+    return loadedData ? loadedData : false;
```

Comment:
why false default? do not do this. add default values as a parameter of the function.

Reply:
Done